RELEASING

```diff
@@ -9,5 +9,7 @@ Releasing shopify_python_api
     git tag -m "Release X.Y.Z" vX.Y.Z
 6. Push the changes to github
     git push --tags origin master
-7. Upload the source package to pypi
-    python setup.py sdist upload
+7. Configure your credentials for pypi in $HOME/pypi.rc
+    (see https://docs.python.org/2/distutils/packageindex.html#the-pypirc-file for instructions)
```

Comment:
Repository must be `https://pypi.python.org/pypi` which is different from the `https://www.python.org/pypi` default mentioned on that page.